engine/glfw/js/library_glfw.js

```diff
@@ -210,6 +210,13 @@ var LibraryGLFW = {
     onKeydown: function(event) {
       if (!GLFW.isCanvasActive(event)) { return; }
 
+      switch(event.keyCode) {
```

Comment:
moved from dmloader.js

Reply:
What _is_ the purpose of this code? 🤔 
Would be good with a code comment I think.

Reply:
Added a comment!

The purpose is to prevent navigation/scrolling inside the page using arrow keys and space (page-down) while the canvas is the active element